File: recipe-site/app/_models/Bounty.ts
import { BountyStatus } from "../_constants/Status";

export default class Bounty {
    bountyId!: number;
    title: string;
    description: string;
    status: BountyStatus;
    categoryId: number;
    expirationDate: string;

    constructor(bountyId: number, title: string, description: string, status: BountyStatus, categoryId: number, expirationDate: string) {
        this.bountyId = bountyId;
        this.title = title;
        this.description = description;
        this.status = status;
        this.categoryId = categoryId;
        this.expirationDate = expirationDate;
    }
}

File: java-recipes/src/main/java/com/recipes/models/Bounty.java
package com.recipes.models;

import java.time.LocalDate;

import com.recipes.enums.BountyStatus;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class Bounty {
	private Integer bountyId;
	private String title;
	private String description;
	private BountyStatus status;
	private Integer categoryId;
	private LocalDate expirationDate;
}
